README.md

```diff
@@ -25,34 +25,38 @@ Each tagged/released version of `hapi-fhir-jpaserver` is built as a Docker image
 
 ```
 docker pull hapiproject/hapi:latest
-docker run -p 8080:8080 -e "spring.batch.job.enabled=false" hapiproject/hapi:tagname
+docker run -p 8080:8080 -e "spring.batch.job.enabled=false" hapiproject/hapi:latest
 ```
 
-This will run the docker image with the default configuration, mapping port 8080 from the container to port 8080 in the host. Once running, you can access `http://localhost:8080/hapi-fhir-jpaserver/` in the browser to access the HAPI FHIR server's UI.
+This will run the docker image with the default configuration, mapping port 8080 from the container to port 8080 in the host. Once running, you can access `http://localhost:8080/hapi-fhir-jpaserver/` in the browser to access the HAPI FHIR server's UI or use `http://localhost:8080/hapi-fhir-jpaserver/fhir/` as the base URL for your REST requests.
```

Comment:
This would need to be modified to from `http://localhost:8080/hapi-fhir-jpaserver/fhir/` to `http://localhost:8080/fhir/ ` based on the latest changes

Reply:
@psavva changed